conf/modules/annotate.config

```diff
@@ -17,9 +17,9 @@ process {
 
     // SNPEFF
     if (params.tools && (params.tools.split(',').contains('snpeff') || params.tools.split(',').contains('merge'))) {
-        withName: 'SNPEFF' {
+        withName: 'SNPEFF_SNPEFF' {
```

Comment:
I am thinking we should start announcing renames in the changelog. So many people rely on this for custom configs etc.

Reply:
Good point

Reply:
especially for new release